### DISCUSSION TITLE
[Question]How to let the null value of line chart to  be "0"

### DISCUSSION THREAD

**Discussion by riceliu** [apache/superset#18288] [Q&A / Help]
As flowing screenshot, how can I make the null value to be zero value?

![image](https://user-images.githubusercontent.com/5362719/113251751-67bcc780-92f5-11eb-8ea3-0bd9b728fa90.png)

**Comment by zhaoyongjie** [comment]
Hey, @riceliu 

define a adhoc metric by follow snippet
```
sum(case ham is null then 0 else ham end)
```

**Comment by srinify** [comment]
Did this work for you @riceliu ? If so, let's close this issue out :)
